docs/training/apis-howto/overview.rst

```diff
@@ -1,32 +1,33 @@
-###############
- Training APIs
-###############
+#####################
+ Training API Guides
+#####################
 
-This section describes how to use the training APIs and contains the API reference information.
+The Training API Guides describe how to use the Determined training APIs. 
+Each API guide contains a link to its corresponding API reference.
 
-Determined leverages specific APIs for each Deep Learning framework. In general, users convert their
+Determined leverages specific APIs for each Deep Learning framework. In general, you convert your
 existing training code by subclassing a Trial class and implementing methods that advertise
-components of the user's model - e.g., model architecture, data loader, optimizer, learning rate
-scheduler, callbacks, etc. This is called the Trial definition and by structuring their code in this
-way, Determined is able to run the training loop and provide advanced training and model management
+components of your model - e.g., model architecture, data loader, optimizer, learning rate
```

Comment:
reword:

> subclassing a Trial class and implementing methods that advertise components of your model - e.g., model architecture, data loader, optimizer, learning rate scheduler, callbacks, etc.

To

> subclassing a Trial class and implementing methods that define to each component of training - e.g., model architecture, data loader, optimizer, learning rate scheduler, callbacks, etc.

Reason: The word "advertising" suggests a specific kind of software architecture that is completely unrelated to what is going on in a Trial class.

Reply:
rewritten